src/python/pants/backend/python/subsystems/python_tool_base.py

```diff
@@ -110,7 +110,11 @@ def register_options(cls, register):
                     "To use a custom lockfile, set this option to a file path relative to the "
                     f"build root, then run `./pants generate-lockfiles "
                     f"--resolve={cls.options_scope}`.\n\n"
-                    ""
+                    "Lockfile generation currently does not wire up the `[python-repos]` options. "
+                    "If lockfile generation fails, you can set this option to a manually "
```

Comment:
This may be a bit clearer:

> "you can produce a lockfile with a tool such as `pip-compile` or `pip freeze` and set this option to the location of that lockfile"